static/src/inline-svgs/membership/videoCaption.svg

```diff
@@ -0,0 +1,4 @@
+
+<svg viewBox="92 875 134 69" preserveAspectRatio="none">
```

Comment:
The viewbox doesn't look right here. It should almost always start with two `0`s. Also, it is normal to have a width and height. see: https://github.com/guardian/frontend/blob/master/docs/inline-svgs.md